app/pages/sections/catalogue-solutions/select/recipient/contextCreator.js

```diff
@@ -2,24 +2,28 @@ import manifest from './manifest.json';
 import { baseUrl } from '../../../../../config';
 import { getSectionErrorContext } from '../../../getSectionErrorContext';
 
-const generateRecipientOptions = ({ recipients }) => (
-  recipients.map(recipient => ({
+const generateRecipientOptions = ({ recipients, selectedRecipientId }) => {
+  const recipientsMap = recipients.map(recipient => ({
     value: recipient.odsCode,
     text: `${recipient.name} (${recipient.odsCode})`,
-  }))
-);
+    checked: selectedRecipientId === recipient.odsCode,
```

Comment:
if you could do it like this please
`checked: selectedRecipientId === recipient.odsCode ? true : undefined,`
that way it'll only put the key and checked as true